public/app/features/alerting/unified/home/GettingStarted.tsx

```diff
@@ -3,23 +3,8 @@ import React from 'react';
 import SVG from 'react-inlinesvg';
 
 import { GrafanaTheme2 } from '@grafana/data';
-import { EmbeddedScene, SceneFlexLayout, SceneFlexItem, SceneReactObject } from '@grafana/scenes';
 import { useStyles2, useTheme2, Stack, Text, TextLink } from '@grafana/ui';
 
-export const getOverviewScene = () => {
```

Comment:
(I need to learn more about Scenes in general, where can I do that? I don't understand the impact of removing this here)

Reply:
I guess this is the best place https://grafana.com/developers/scenes/